contrib/devtools/Dockerfile

```diff
@@ -26,8 +26,8 @@ ENV GOLANG_PROTOBUF_VERSION=1.28.1 \
 
 RUN go install github.com/cosmos/cosmos-proto/cmd/protoc-gen-go-pulsar@latest && \
   go install google.golang.org/protobuf/cmd/protoc-gen-go@v${GOLANG_PROTOBUF_VERSION} && \
-  go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v${GRPC_GATEWAY_VERSION} \
-  github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@v${GRPC_GATEWAY_VERSION} && \
+  go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v${GRPC_GATEWAY_VERSION} && \
+  go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2@latest && \
```

Comment:
Is it ok to use latest here?

Reply:
we should use a tagged version.

Reply:
@dadamu could you use a tag here please

Reply:
Updated: https://github.com/cosmos/cosmos-sdk/pull/20448/commits/4c29ad93f23c79ff0b56c50a35e1d3f11da44234